src/jdk.httpserver/share/classes/com/sun/net/httpserver/Filter.java

```diff
@@ -26,17 +26,21 @@
 package com.sun.net.httpserver;
 
 import java.io.IOException;
-import java.util.*;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.util.List;
+import java.util.ListIterator;
 
 /**
  * A filter used to pre- and post-process incoming requests. Pre-processing occurs
  * before the application's exchange handler is invoked, and post-processing
  * occurs after the exchange handler returns.  Filters
- * are organised in chains, and are associated with HttpContext instances.
- * <p>
- * Each Filter in the chain, invokes the next filter within its own
- * doFilter() implementation. The final Filter in the chain invokes the applications
- * exchange handler.
+ * are organised in chains, and are associated with {@link HttpContext} instances.
+ *
+ * <p> Each {@Filter} in the chain, invokes the next filter within its own
```

Comment:
\@Filter Is that supposed to be {@code Filter} or {@link Filter}?

Reply:
Ah! I missed that. Well spotted Michael. Should be `{@code Filter}` of course.

Reply:
Thanks for spotting this, Michael. I've rectified this in the latest commit (https://github.com/openjdk/jdk/pull/429/commits/c4a654832e81740543313e2a86a245865f3283b9).